src/modules/notifications/NotificationConstants.ts

```diff
@@ -36,34 +45,37 @@ const NotificationConstants = {
     },
     NotificationSetting: {
         General: {
-            encountered_shiny: new BooleanSetting('notification.encountered_shiny', 'Encountered a shiny Pokémon', true),
-            quest_ready_to_complete: new BooleanSetting('notification.quest_ready_to_complete', 'Quest is ready to be completed', true),
-            underground_energy_full: new BooleanSetting('notification.underground_energy_full', 'Mining energy reached maximum capacity', true),
-            event_start_end: new BooleanSetting('notification.event_start_end', 'Event start/end information', true),
-            gym_won: new BooleanSetting('notification.gym_won', 'Gym leader defeated', true),
-            offline_earnings: new BooleanSetting('notification.offline_earnings', 'Offline earnings', true),
+            encountered_shiny: new NotificationSetting('notification.encountered_shiny', 'Encountered a shiny Pokémon', true, false, NotificationWarnOnBlocked),
+            quest_ready_to_complete: new NotificationSetting('notification.quest_ready_to_complete', 'Quest is ready to be completed', true, false, NotificationWarnOnBlocked),
+            underground_energy_full: new NotificationSetting('notification.underground_energy_full', 'Mining energy reached maximum capacity', true, false, NotificationWarnOnBlocked),
+            event_start_end: new NotificationSetting('notification.event_start_end', 'Event start/end information', true, false, NotificationWarnOnBlocked),
+            gym_won: new NotificationSetting('notification.gym_won', 'Gym leader defeated', true, false, NotificationWarnOnBlocked),
+            offline_earnings: new NotificationSetting('notification.offline_earnings', 'Offline earnings', true, false, NotificationWarnOnBlocked),
+            achievement_complete: new NotificationSetting('notification.achievement_complete', 'Achievement complete', true, false, NotificationWarnOnBlocked),
+            new_catch: new NotificationSetting('notification.new_catch', 'New Pokémon/shiny captured', true, false, NotificationWarnOnBlocked),
+            battle_frontier: new NotificationSetting('notification.battle_frontier', 'Battle Frontier', true, false, NotificationWarnOnBlocked),
```

Comment:
Could we disable the option for these to be turned off in game, but still able to enable desktop notifications, we want these notifications to always be displayed.

Reply:
done